docs/standard/design-guidelines/arrays.md

```diff
@@ -8,7 +8,8 @@ helpviewer_keywords:
   - "empty arrays"
 ms.assetid: 66a1b3d8-6f3f-4715-b235-e1ff95e32d8e
 ---
-# Arrays
+# Arrays design guidelines
```

Comment:
Do you want me to add this suffix to all design guidelines articles for consistency?

Reply:
I think no, because this is copyrighted content.